builtin/types/user_deployer_whitelist/user_deployer_whitelist.proto

```diff
@@ -41,14 +41,16 @@ message UserState {
 message UserDeployerState {
     Address address = 1;
     repeated DeployerContract contracts = 2;
+    TierID tierId = 3;
```

Comment:
`tierId` -> `tier_id`, and add the custom name tag to make it `TierID` in golang.

Reply:
done